.circleci/config.yml

```diff
@@ -54,8 +54,40 @@ jobs:
       - store_artifacts:
           path: *TEST_RESULTS_PATH
 
+  go-smb-test:
+    working_directory: /go-getter
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: install dependencies
+          command: |
+            apk add --no-cache \
+              py-pip=9.0.0-r1
+            pip install \
+              docker-compose==1.12.0
+
+      - run:
+          name: build and start smb server and gogetter containers
+          command: |
+            docker-compose build
+            docker-compose up -d
+
+      - run:
+          name: wait for containers to start
+          command: sleep 60
+
+      - run:
+          name: run smb getter tests
+          command: |
+            docker exec -it samba bash -c "echo 'Hello' > /mnt/file.txt && mkdir -p /mnt/subdir  && echo 'Hello' > /mnt/subdir/file.txt"
```

Comment:
I couldn't create the files in the Dockerfile, this somehow was not working on CircleCi so I had to create them later.